### ISSUE TITLE
requestModule should be separate file?

### ISSUE THREAD

**Issue by mattlub** [FACN1/week5-facn-hosers#21]
https://github.com/FACN1/week5-facn-hosers/blob/master/public/assets/main.js#L1

**Comment by philawsophizing** [comment]
I disagree in this instance, since main.js would be one line if we split the request out.. would be overly modularized/make the file structure unnecessarily complicated. But I would agree in a more complicated app.

**Comment by mattlub** [comment]
in this case you don't really need the module at all, just have a make request function

**Comment by m4v15** [comment]
go away

**Comment by mattlub** [comment]
just trying to help
